README.md

```diff
@@ -64,6 +60,39 @@ by copying it to your local `config` directory or by defining the environment va
 php artisan vendor:publish --provider="Kreait\Laravel\Firebase\ServiceProvider" --tag=config
 ```
 
+### Credentials with JSON files auto-discovered
```

Comment:
```suggestion
### Credentials with JSON files
```

They can be auto-discovered but don't have to be, the details are explained in the paragraph.